tools/cxbe/Xbe.cpp

```diff
@@ -394,7 +395,11 @@ Xbe::Xbe(class Exe *x_Exe, const char *x_szTitle, uint32 x_dwTitleID, uint32 x_d
                        (characteristics & IMAGE_SCN_CNT_CODE))
                         m_SectionHeader[v].dwFlags.bExecutable = true;
 
-                    m_SectionHeader[v].dwFlags.bPreload = true;
+                    char *name = (x_Exe->m_SectionHeader_longname[v].m_longname)
+                                     ? x_Exe->m_SectionHeader_longname[v].m_longname
+                                     : (char *)x_Exe->m_SectionHeader[v].m_name;
+                    m_SectionHeader[v].dwFlags.bPreload =
+                        (strcmp(name, ".debug") && strncmp(name, ".debug_", 7));
```

Comment:
Why not just `strncmp(name, ".debug", 6)`?

Reply:
`.debugTest` is not a real debug section name while `.debug` is and anything starting with `.debug_` is

Reply:
Which `.debugTest`? That is not a section I've ever seen nxdk produce (and nothing else should because the dot prefix is reserved for system sections)

Reply:
It could be user generated. I only did it for completeness and it's not like it would hurt performance terribly. The more performance-oriented option would be to check if the first 6 matched `.debug` and then check if the 7th was `\0` or `_`

Reply:
> It could be user generated

The dot prefix is reserved. I don't care about performance here as much as clean code.